ONSdigital/rm-case-service
# Add script to remove multiple B cases for a case group

# Motivation and Context

Add script to remove all instances of multiple B cases for case groups. It is unnecessary to have multiple B cases per case group as only the most recent one will be used. 

# What has changed




# How to test?

Add test data for multiple B cases in a case group then run script check most recent is still there.



# Links

[Trello] https://trello.com/c/NKpllwnm
